<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Ayuda</class>
 <widget class="QDialog" name="Ayuda">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>597</width>
    <height>289</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ventana De Ayuda</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="2">
    <widget class="QPushButton" name="pushButton_Salir">
     <property name="text">
      <string>Salir</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_Nuevo">
      <attribute name="title">
       <string>Tab 1</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QPlainTextEdit" name="plainTextEdit_Nuevo"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_Abrir">
      <attribute name="title">
       <string>Tab 2</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QPlainTextEdit" name="plainTextEdit_Abrir"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_Guardar">
      <attribute name="title">
       <string>Page</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <widget class="QPlainTextEdit" name="plainTextEdit_Guardar"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_Imprimir">
      <attribute name="title">
       <string>Page</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="0" column="0">
        <widget class="QPlainTextEdit" name="plainTextEdit_Imprimir"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_Ayuda">
      <attribute name="title">
       <string>Page</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_6">
       <item row="0" column="0">
        <widget class="QPlainTextEdit" name="plainTextEdit_Ayuda"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_PuertoSerie">
      <attribute name="title">
       <string>Page</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_7">
       <item row="0" column="0">
        <widget class="QPlainTextEdit" name="plainTextEdit_PuertoSerie"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_Ruta">
      <attribute name="title">
       <string>Page</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_8">
       <item row="0" column="0">
        <widget class="QPlainTextEdit" name="plainTextEdit_Ruta"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_Salir">
      <attribute name="title">
       <string>Page</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_9">
       <item row="0" column="0">
        <widget class="QPlainTextEdit" name="plainTextEdit_Salir"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPlainTextEdit" name="plainTextEdit">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>34</height>
      </size>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="plainText">
      <string>(C) 2017 Alejandro Herme. Colaboracion GUI: Gustavo PÃ©rez. Todos los derechos reservados
adhsoft@gmail.com</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_Salir</sender>
   <signal>clicked()</signal>
   <receiver>Ayuda</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>346</x>
     <y>383</y>
    </hint>
    <hint type="destinationlabel">
     <x>204</x>
     <y>210</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
